modules/rpc/yarpc.go

```diff
@@ -249,7 +269,24 @@ func newYARPCModule(
 	module.config.inboundMiddleware = inboundMiddlewareFromCreateInfo(mi)
 	module.config.onewayInboundMiddleware = onewayInboundMiddlewareFromCreateInfo(mi)
 
-	_controller.addConfig(module.config)
+	// Try to resolve a controller first
+	// TODO(alsam) use dig options when available, because we can overwrite the controller in case of multiple
+	// modules registering a controller.
+	if err := dig.Resolve(&module.controller); err != nil {
```

Comment:
No, because it can be a valid error: no one registered a controller, and this is what we are going to do in the next line.